flocker/acceptance/testtools.py

```diff
@@ -282,10 +288,10 @@ def no_containers(cluster):
     getting.addCallback(lambda cluster:
                         loop_until(lambda: no_containers(cluster)))
 
-    def clean_zfs(_):
+    def clean(_):
         for node in reachable_nodes:
             _clean_node(test_case, node)
-    getting.addCallback(clean_zfs)
+    getting.addCallback(clean)
```

Comment:
Merging forward there's an if statement now, so I assume this will be dealt with by the configurable backend branch.